src/utils/index.js

```diff
@@ -117,7 +117,6 @@ exports.flattenData = (data, xAccessor, yAccessor) => {
         },
         d: item,
         id: series.name + j,
-        series: series,
```

Comment:
Ah ok, so this is intended. I thought it might slow down the charts if you have a lot of points...